Update User Guide - Multi-band color mapping for COG layers and Raster layer section (#10137)
geosolutions-it/MapStore2: docs/user-guide/layer-settings.md

<img src="../img/layer-settings/single_band_style.jpg" class="ms-docimage">

#### RGB bands

With the **RGB bands** the user can customize the three **Channels** of the layer, if they are available, and for each one selecting the **Contrast enhancement** choosing between `None`, `Normalize` and `Histogram` and the change the **Opacity**. Take a look at the following example.

<img src="../img/layer-settings/rgb_band_style.jpg" class="ms-docimage">

#### Pseudo color

With the **Pseudo color** the user has the possibility to customize the *Single band* with a *Color ramp*. In this case, in addition to choosing **Channels**, **Contrast enhancement** and **Opacity**, the user can select **Color ramp**, enable **Reverse order** and the **Continuous Colors**, choose the **Color map type** (`Ramp`, `Intervals` and `Values`), the **Method** (`Quantile`, `Natural breaks`, `Equal interval` and `Unique interval`) and modify the **Intervals**. Take a look at the following example.

<img src="../img/layer-settings/pseudo_color_style.jpg" class="ms-docimage">

#### Styling for COG layer

> With the *Visual Style Editor*, the editor has the ability to customize the style of **COG layer**:

Reviewer: ```suggestion
With the *Visual Style Editor*, the editor has the ability to customize the style of **COG layers**:
```

<img src="../img/layer-settings/cog-layer_style.jpg" class="ms-docimage">

> Here the editor can associate, when the *Band styling* is enable,  the **Channels** (`Red`, `Green`, `Blue` and `Alpha`) to the **Bands** of the COG layer and set the minimum (**Min**) and maximum (**Max**) source data value. Take a look at the following example.

Reviewer: ```suggestion
Enabling the *Band styling* section, here the editor can associate the **Channels** (`Red`, `Green`, `Blue` and `Alpha`) to the **Bands** of the COG layer and set the minimum (**Min**) and maximum (**Max**) source data value. Take a look at the following example.
```
